packages/popover/src/Popover/Popover.stories.jsx

```diff
@@ -334,3 +352,22 @@ export const RendersOnSpecificArea = {
     await new Promise((resolve) => setTimeout(resolve, 2000))
   },
 }
+
+export const DisableFloating = (args) => {
+  return (
+    <Flex m={200}>
+      <Popover
+        color='primary'
+        renderContent={LongTextContent}
+        placement='bottom'
+        ariaLabel='Default Popover'
+        width={330}
+        disableFloating={args.disableFloating}
```

Comment:
Can you spread the `args` as props on `<Popover>`? That's how Storybook forwards the args to the component in the story.

Reply:
Will do!